browser/extensions/api/brave_rewards_api.cc

```diff
@@ -85,6 +86,14 @@ RewardsPanelCoordinator* GetPanelCoordinator(ExtensionFunction* function) {
 namespace extensions {
 namespace api {
 
+BraveRewardsIsSupportedFunction::~BraveRewardsIsSupportedFunction() = default;
```

Comment:
do we need to know specifically that it's disabled by policy and not just disabled in general (profile type, etc...)?

Reply:
for instance should `brave_rewards::features::kBraveRewards` disabled also apply here?